homeassistant/components/weatherflow_cloud/strings.json

```
@@ -56,6 +56,34 @@
       "lightning_strike_last_epoch": {
         "name": "Lightning last strike"
       },
+      "precip_accum_last_1hr": {
+        "name": "Rain last 1 hr"
+      },
+
+      "precip_accum_local_day": {
+        "name": "Precip today"
+      },
+      "precip_accum_local_day_final": {
+        "name": "Nearcast precip today"
+      },
+      "precip_accum_local_yesterday": {
+        "name": "Precip yesterday"
+      },
+      "precip_accum_local_yesterday_final": {
+        "name": "Nearcast precip yesterday"
+      },
+      "precip_analysis_type_yesterday": {
+        "name": "Precip type yesterday"
+      },
+      "precip_minutes_local_day": {
+        "name": "Precip min today"
+      },
+      "precip_minutes_local_yesterday": {
+        "name": "Precip min yesterday"
+      },
+      "precip_minutes_local_yesterday_final": {
+        "name": "Nearcast precip min yesterday"
+      },
```

These three already show up with "min" as the unit of measurement. So it does not have to be repeated in the entity name. In addition there is a big risk of "min" being (machine) translated as "minimum".

```suggestion
      "precip_minutes_local_day": {
        "name": "Precip duration today"
      },
      "precip_minutes_local_yesterday": {
        "name": "Precip duration yesterday"
      },
      "precip_minutes_local_yesterday_final": {
        "name": "Nearcast precip duration yesterday"
      },
```